### PR TITLE
i18n(fr): update `guides/authoring-content.md`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR updates the french version of the `guides/authoring-content.md` page with the changes of #742.

### CODE REVIEW

File: docs/src/content/docs/fr/guides/authoring-content.md [hunk 1]
```diff
@@ -179,12 +179,12 @@ Vos utilisateurs peuvent être plus productifs et trouver votre produit plus fac
 > Les guillemets sont indiqués par un `>` au début de chaque ligne.
 ```
 
-## Code blocks
+## Blocs de code
 
-Un bloc de code est indiqué par un bloc avec trois crochets <code>```</code> au début et à la fin. Vous pouvez indiquer le langage de programmation utilisé après les premiers crochets.
+Un bloc de code est indiqué par un bloc avec trois accents graves <code>```</code> au début et à la fin. Vous pouvez indiquer le langage de programmation utilisé après les premiers accents graves.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/authoring-content.md:184]: [Reference](https://fr.wiktionary.org/wiki/backtick) for "accent grave"

**Review by delucis — APPROVED** [review]
Thanks for the update @HiDeoo! Translation and code blocks LGTM 🙌

**Comment by HiDeoo** [comment]
PR updated with the changes of #1119

**Review by delucis — APPROVED** [review]
Re-approving with the updated links!

**Pull request merged**